Dockerfile

```diff
@@ -31,6 +31,8 @@ RUN a2enmod rewrite proxy proxy_fcgi headers
 RUN sed -ie 's/# en_US.UTF-8 UTF-8/en_US.UTF-8 UTF-8/g' /etc/locale.gen
 RUN sed -ie 's/# en_GB.UTF-8 UTF-8/en_GB.UTF-8 UTF-8/g' /etc/locale.gen
 RUN sed -ie 's/# de_DE.UTF-8 UTF-8/de_DE.UTF-8 UTF-8/g' /etc/locale.gen
+RUN sed -ie 's/# hi_IN UTF-8/hi_IN UTF-8/g' /etc/locale.gen
```

Comment:
somehow, the line `# hi_IN.UTF-8 UTF-8` didn't exist in the /etc/locale.gen, so I went with the option that was in there... Not sure why, though...

Reply:
?

Reply:
The /etc/locale.gen file we change here, looked something like this:
```
[...]
# de_DE ISO-8859-1       
# de_DE.UTF-8 UTF-8        
[...]
# en_GB ISO-8859-1
# en_GB.ISO-8859-15 ISO-8859-15
# en_GB.UTF-8 UTF-8
[...]
# es_ES ISO-8859-1
# es_ES.UTF-8 UTF-8
[...]
# hi_IN UTF-8
[...]
```

Didn't know why `hi_IN` is different from the other UTF-8, but that's why the code here is different too. As @inyono told me: It's probably because there aren't hindi symbols in ISO-8859-1.